### PR TITLE
0.9.12 prerelease2

### PR DESCRIPTION
**Pull request by Scarjit**
Supersedes #1524

### CODE REVIEW

**Comment by DaruZero** [comment]
we need to add backwards compatibility for redis

File: deployment/helm-repo/index.yaml [hunk 1]
```diff
@@ -286,11 +344,11 @@ entries:
     - https://github.com/united-manufacturing-hub/united-manufacturing-hub
     type: application
     urls:
-    - https://repo.umh.app/united-manufacturing-hub-0.9.7.tgz
+    - https://0-9-12-prerelease2.united-manufacturing-hub.pages.dev/united-manufacturing-hub-0.9.7.tgz
```
Comment (JeremyTheocharis) [inline deployment/helm-repo/index.yaml:347]: remember to change back

File: deployment/united-manufacturing-hub/Chart.yaml [hunk 2]
```diff
@@ -32,15 +32,15 @@ type: application
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 
-version: 0.9.11
+version: 0.9.12-prerelease2
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/Chart.yaml:35]: remember to change

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

**Comment by JeremyTheocharis** [comment]
Please also look at the automated tests, which are failing atm

**Pull request closed without merging**